tools/gen_selectedops.py

```diff
@@ -1,80 +1,136 @@
-import os
+#!/usr/bin/env python3
```

Comment:
FYI, probably don't need to include the gen_selectedops.py changes in your branch. it should functionally be the same as before.